recipes/gobject-introspection/all/conandata.yml

```diff
@@ -1,19 +1,7 @@
 sources:
+  "1.78.1":
+    url: "https://download.gnome.org/sources/gobject-introspection/1.78/gobject-introspection-1.78.1.tar.xz"
+    sha256: "bd7babd99af7258e76819e45ba4a6bc399608fe762d83fde3cac033c50841bb4"
   "1.72.0":
-    sha256: "02fe8e590861d88f83060dd39cda5ccaa60b2da1d21d0f95499301b186beaabc"
     url: "https://download.gnome.org/sources/gobject-introspection/1.72/gobject-introspection-1.72.0.tar.xz"
-  "1.70.0":
-    url: "https://download.gnome.org/sources/gobject-introspection/1.70/gobject-introspection-1.70.0.tar.xz"
-    sha256: "902b4906e3102d17aa2fcb6dad1c19971c70f2a82a159ddc4a94df73a3cafc4a"
-  "1.69.0":
-    url: "https://download.gnome.org/sources/gobject-introspection/1.69/gobject-introspection-1.69.0.tar.xz"
-    sha256: "c668cbe4a3aad7539e2cf669ab576ce7fbadac6890472f4095ca215dbbebee99"
-  "1.68.0":
```

Comment:
The version 1.68.0 is still used by other packages in master branch:

- gst-libav
- gst-plugins-bad
- gst-plugins-good


The `gst-plugins-base` uses 1.70.0

Reply:
It's easier to bump these than add support for older versions of `gobject-introspection`, though. I dropped them due to issues on Windows, if I remember correctly.

Reply:
I agree, please, update those packages first, then we can update it here.